posts/how-to-install-memgraph-and-memgraph-lab-with-docker-on-macOS.md

```diff
@@ -0,0 +1,233 @@
+
+To do this, first, open a new window or tab in your command line. The window or tab you used to run the Memgraph server can't execute another command. Simply run this command on a new window or tab:
+
+```
+docker ps
+```
+
+You should then see an output similar to the following:
+
+```
+CONTAINER ID        IMAGE           COMMAND                  CREATED         ...
+d1d9d4ce649f        memgraph        "/usr/lib/memgraph/m…"   2 seconds ago   ...
+```
+
+The value we're interested in is the one in the `CONTAINER ID` column. In the example output above, it's `d1d9d4ce649f`.
+
+Now, use the container ID to retrieve the IP address of the container. Run the command below, at the same time, replacing `CONTAINER_ID` with your container ID:
+
+```
+docker inspect -f '' CONTAINER_ID
+```
+
+The command above will return a long JSON string that contains various properties that describe the container. Here's a truncated version of it:
+
+```
+[
+  {
+    "Id": "d1d9d4ce649fc5d78dee28eb946ac6f5b352f6ef1ee05eb6518bdec4061a55ef",
+    .......
+    "NetworkSettings": {
+        "Gateway": "172.17.0.1",
+        "IPAddress": "172.17.0.2",
+        ......
+    }
+  }
+]
+```
+
+Inside the JSON string, look for the value of the key named `IPAddress` as this is the IP address we are looking for. Yours will vary, but, in our case, it's `172.17.0.2`.
+
+Next, run the `mg_client` with this command, and at the same time, correctly replacing `HOST` with your IP address:
+
+```
+docker run -it --entrypoint=mg_client memgraph --host HOST
+```
+
+If the `mg_client` can connect to the server, you'll receive a prompt similar to this:
+
+```
+mg_client 0.14.1
+Type :help for shell usage
+Quit the shell by typing Ctrl-D(eof) or :quit
+Connected to 'memgraph://172.17.0.2:7687'
+memgraph>
+```
+
+You are now ready to execute Cypher queries on the server. Each query needs to end with the `;` (_semicolon_) character.
+
+<p id="first-query">Let's begin by executing this first query:</p>
+
+```
+CREATE (u:User {name: "Alice"})-[:Likes]->(m:Software {name: "Memgraph"});
+```
+<p id="query-explain">The above will create 2 nodes in the database, one labeled "User" with the name "Alice" and the other labeled "Software" with the name "Memgraph". It will also create a relationship that "Alice" likes "Memgraph".</p>
+
+Next, find the created nodes and relationships by executing this second query:
+
+```
+MATCH (u:User)-[r]->(x) RETURN u, r, x;
+```
+
+The query returns the following result:
+```
++--------------------------------+--------------------------------+--------------------------------+
+
+| u  | r  | x  |
+
++--------------------------------+--------------------------------+--------------------------------+
+
+| (:User {name: "Alice"})  | [Likes]  | (:Software {name: "Memgraph"}) |
+
++--------------------------------+--------------------------------+--------------------------------+
+
+1 row in set (0.038 sec)
+```
```

Comment:
This is going to render badly. Options are:
1) Make a screenshot.
2) Remove ``` to render the standard table (`1 row...` is going to be a problem).
3) Format the code block to look nicer.

Reply:
I'm not sure what do you mean here.
1. This option looks bad because the colors will not match the rest of the blog post
2. I did this and all it does is remove the grey background. It becomes a little confusing for the reader.
3. Not sure what you have in mind here.

Unless we add a new rendering format for terminal display blocks, this seems to be the best option for now.

Reply:
Aha, I see now why formatting is broken. There are unnecessary empty lines + the separator lines `+---...+---...` should be the same width as the longest line. The same applies to the other pull request.

Reply:
I fixed it the way you suggested. Is this good now?